example_model/Dockerfile

```diff
@@ -11,9 +11,10 @@
 
 FROM telestoai/model-api-base:latest
 
+WORKDIR /root
+
 # Copying files to the image
-COPY example_model /
-COPY telesto /telesto
+COPY example_model .
```

Comment:
You have updated the build command and set the build context to `example_model`. Shouldn't this be
```
COPY * /
```
?

Reply:
Yep, fixed it.